src/app/enums/organization-types.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum OrganizationType {
+  Buyer = 'Buyer',
+  Developer = 'Developer',
+  ApiUser = 'ApiUser',
+  Admin = 'Admin',
+}
```

Comment:
why did you have to create this enum again yet we had another one ?
```
export enum OrganizationType {
  User = 'User',
  DeviceOwner = 'DeviceOwner',
  OrganizationAdmin = 'OrganizationAdmin',
  Buyer = 'Buyer',
  Admin = 'Admin',
  SubBuyer = 'SubBuyer',
  Developer = 'Developer',
  ApiUser = 'ApiUser',
}
```
please consider using what we have already to avoid code duplicatiion